.github/workflows/dev-docs.yml

```diff
@@ -25,9 +25,9 @@ jobs:
             bids_postprocess:
               - 'xnat_tools/bids_postprocess.py'
       - name: Set up Python 3.7 
-        uses: actions/setup-python@v2 
-        with: 
-          python-version: '3.7.5' 
+        uses: actions/setup-python@v4
+        with:
```

Comment:
I think by default it reads the version from `.python-version` and therefore the whole `with` section isn't needed. That's how I interpreted their info

Reply:
I noticed without specifying: 

`with:
        python-version-file: '.python-version`

setup-python was downloading 3.11 (latest), not detecting the version file by default. This change was made in the following commit.